examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -286,9 +286,9 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 				.json({ message: "Missing parameter externalTaskListId in request url" });
 		}
 		externalDataSource.fetchData(externalTaskListId).then(
-			(response) => {
-				// eslint-disable-next-line @typescript-eslint/no-base-to-string
-				const responseBody = JSON.parse(response.body.toString()) as Record<
+			async (response) => {
```

Comment:
Now that the inner lambda is async, I don't believe we are correctly catching any errors that occur here. It might be better to convert this to a `.then(...)` pattern with its own catch lambda.